pkg/workceptor/workceptor.go

```diff
@@ -16,17 +17,28 @@ import (
 
 	"github.com/ansible/receptor/pkg/certificates"
 	"github.com/ansible/receptor/pkg/controlsvc"
+	"github.com/ansible/receptor/pkg/logger"
 	"github.com/ansible/receptor/pkg/netceptor"
 	"github.com/ansible/receptor/pkg/randstr"
 	"github.com/ansible/receptor/pkg/utils"
 	"github.com/golang-jwt/jwt/v4"
 )
 
+// NetceptorForWorkceptor is a interface to decouple workceptor from netceptor.
+// it includes only the functions that workceptor uses.
+type NetceptorForWorkceptor interface {
```

Comment:
One final suggestion from Shane seems to be the winner, `NetceptorForWorkceptor` will be the new standard we set.